Automattic/vip-cli
# Move vip-import-sql to the CLI command line output

## Description

Wires up the automated sql import command with a description and usage example.

## Steps to Test

1. Check out PR on your sandbox and `npm run build`
2. `node ./dist/bin/vip-import.js`
3. `node ./dist/bin/vip-import-sql.js`

Note that there is descriptive output.


---
_This PR was started by: [git wf pr](https://github.com/groupon/git-workflow/releases/tag/v1.3.2)_
